test/650_proxy_env_test.sh

```diff
@@ -28,4 +28,10 @@ weave_on $HOST1 stop
 weave_on $HOST1 launch-proxy -H tcp://0.0.0.0:12375 -H unix:///var/run/weave/weave.sock
 check
 
+# Check we can use weave env/config with unix -Hs specified
+weave_on $HOST1 stop
+weave_on $HOST1 launch-proxy -H unix:///var/run/weave/weave.sock
+assert_raises 'run_on '$HOST1' "eval $(weave env) ; docker '$CMD'"'
+assert_raises 'run_on '$HOST1' "docker $(weave config) '$CMD'"'
```

Comment:
Yeah, am having some issues with the quotation getting stuff to execute in the right place.